doc/classes/Array.xml

```diff
@@ -383,8 +383,7 @@
 			<param index="0" name="position" type="int" />
 			<param index="1" name="value" type="Variant" />
 			<description>
-				Inserts a new element at a given position in the array. The position must be valid, or at the end of the array ([code]pos == size()[/code]).
-				[b]Note:[/b] This method acts in-place and doesn't return a value.
+				Inserts a new element at a given position in the array and returns an Error value. The position must be valid, or at the end of the array ([code]pos == size()[/code]).
```

Comment:
"and returns an Error value" while technically true, doesn't sound right to me. It makes it sound like it returns errors all the time, and your edit also doesn't link to the possible values. You should be more explicit.

```suggestion
				Inserts a new element at a given position in the array. The position must be valid, or at the end of the array ([code]pos == size()[/code]). Returns [constant OK] on success, or one of the other [enum Error] values if the insertion failed.
```

Reply:
Could use the wording of `ConfigFile`:
"Returns one of the [enum Error] code constants ([constant OK] on success)."

Reply:
I saw that and didn't like it 🙃 I would update that one as well (but it's not related to this PR and shouldn't be done here).